Documentation/README_ObjectManipulator.md

```diff
@@ -0,0 +1,242 @@
+* *Rotate about grab point*: Rotate object with the hand about the grab point between your thumb and index finger. It should feel as if the object is being held by hand.
+
+#### One hand rotation mode far
+Specifies how the object will behave when it is being grabbed with one hand at distance. These options only work for articulated hands.
+
+* *Rotate about object center*: Rotate object using rotation of the hand, but about the object center point. Useful for inspecting at a distance without the object center moving as the object rotates.
+* *Rotate about grab point*: Rotate object using rotation of the hand, but about the pointer ray hit point. Useful for inspection.
+
+### Two handed manipulation properties
+
+#### Two handed manipulation type
+
+Specifies how two hand manipulation can transform an object. Because this property is a flag, you can select any number of options.
+
+* *Move*: Moving is allowed if selected.
+* *Scale*: Scaling is allowed if selected.
+* *Rotate*: Rotation is allowed if selected.
+
+![Manipulation Handler](../Documentation/Images/ManipulationHandler/MRTK_ManipulationHandler_TwoHanded.jpg)
+
+### Constraints
+
+#### Add constraint
+
+This button allows you to add a constraint component directly from the object manipulator inspector. All constraints in your project should be visible here. See [transform constraints](#transform-constraints) for more info.
+
+#### Go to component
+
+All constraints found on the object wil be listed here with a *Go to component* button. This button will cause the inspector to scroll to the constraint component you selected so that you can configure it. 
+
+### Physics
+
+#### Release behavior
+
+Specify which physical properties a manipulated object should keep upon release. Requires a rigidbody component to be on that object. Because this property is a flag, you can select any number of options.
+
+* *Keep Velocity*: When the object is released, if this option is selected it will keep its linear velocity.
+* *Keep Angular Velocity*: When the object is released, if this option is selected it will keep its angular velocity.
+
+### Smoothing
+
+#### Smoothing active
+Specifies whether smoothing is active.
+
+#### Move lerp time
+Amount of smoothing to apply to the movement. Smoothing of 0 means no smoothing. Max value means no change to value.
+
+#### Rotate lerp time
+Amount of smoothing to apply to the rotation. Smoothing of 0 means no smoothing. Max value means no change to value.
+
+#### Scale lerp time
+Amount of smoothing to apply to the scale. Smoothing of 0 means no smoothing. Max value means no change to value.
+
+### Manipulation events
+
+Manipulation handler provides the following events:
+
+* *OnManipulationStarted*: Fired when manipulation starts.
+* *OnManipulationEnded*: Fires when the manipulation ends.
+* *OnHoverStarted*: Fires when a hand / controller hovers the manipulatable, near or far.
```

Comment:
Nit: Is it also worth covering the event fire order (i.e.HoOverStarted->ManipulationStarted->ManipulationEnded->HoverEnded?)